### DISCUSSION TITLE
Inline Twist for Twister roller coaster

### DISCUSSION THREAD

**Discussion by TheAtomTwister** [OpenRCT2/OpenRCT2#16385] [Feature requests] [answered]
So someone had a great idea to add inline twists to the corkscrew roller coaster, and I'd like to see them on the Twister tracks.  The sprites are used by the flying roller coaster, so this seems like another easy one.
![Flying RC and Twister RC 2](https://user-images.githubusercontent.com/97222164/148363598-10546305-171a-49c8-a5ab-e916be25e5cb.png)
![Flying RC and Twister RC 3](https://user-images.githubusercontent.com/97222164/148363604-5aab45b2-4991-46a6-b0bf-c91e1255db0f.png)
It might also be cool to add to the Flying roller coaster all of the Twister's inversions.  I don't know if the vehicle animations exist for that, but it might be worth a shot.  There may be unused Flying RC animations just waiting for us.
![Flying RC and Twister RC](https://user-images.githubusercontent.com/97222164/148364304-7855c479-b999-4292-897a-3d8b2a8557a1.png)

**Comment by karst** [comment, marked as answer]
I'll look into it :)
